internal/pkg/service/appsproxy/http/router.go

```diff
@@ -174,9 +174,9 @@ func (r *Router) Shutdown() {
 func (r *Router) createConfigErrorHandler(exceptionID string) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 		r.logger.With(attribute.String("exceptionId", exceptionID)).Warn(req.Context(), `application "<proxy.appid>" has misconfigured OAuth2 provider`)
+		appID, _ := req.Context().Value(AppIDCtxKey).(string)
```

Comment:
Adding the app name will be solved in the next task -> [here](https://keboola.atlassian.net/browse/PSGO-499)